.github/CODEOWNERS

```diff
@@ -831,7 +831,7 @@
 #/<NotInRepo>/          @derek1ee
 
 # ServiceLabel: %Search %Service Attention
-#/<NotInRepo>/          @bleroy @tjacobhi @markheff @miwelsh
+#/<NotInRepo>/          @mattgotteiner
```

Comment:
We can remove this section and add service owner at https://github.com/Azure/azure-sdk-for-python/blob/main/.github/CODEOWNERS#L337, right?

Reply:
Yes, though that section is also not formatted correctly.   As written, you are considered the service owner, as it is inferred from the PR path.   I'd suggest breaking this down into two distinct sections, one for PRs one for triage.   This would become something like:

```text
# PRLabel: %Search
/sdk/search/     @xiangyan99

# ServiceLabel: %Search
# AzureSdkOwners: @xiangyan99
# ServiceOwners:  @mattgotteiner
```

If you also needed to have a service owner for the Search management package, you'd have a second block:

```text
# PRLabel: %Search
/sdk/search/     @xiangyan99

# ServiceLabel: %Search
# AzureSdkOwners: @xiangyan99
# ServiceOwners:  @mattgotteiner

# PRLabel: %Search
/sdk/search/azure-resourcemanager-*/   @some-service-owner

# ServiceLabel: %Search %Mgmt
# ServiceOwners:  @some-service-owner

```

Reply:
Currently, Matt has not joined the org so we cannot add him as owner. I talked to him offline and we will update it when he joined the org.

Reply:
I am a member of the org